draft-ietf-tls-dtls13.md

```diff
@@ -1660,17 +1674,6 @@ and receiving a NewConnectionId message in response can only decide that the New
 message should be treated through an independent state machine after inspecting the handshake
 message type.
 
-##  CertificateVerify and Finished Messages
-
-CertificateVerify and Finished messages have the same format as in
-TLS 1.3.  Hash calculations include entire handshake messages, including
```

Comment:
I mostly assume no one implemented the literal reading of this, where CertificateVerify/Finished do not have the fragment-specific fields

Reply:
Well that's what we are telling people to do now.